docs/conf.py

```diff
@@ -321,6 +321,7 @@ def _custom_edit_url(
 # A dictionary of values to pass into the template engine's context for all pages.
 html_context = {
     "default_mode": "light",
+    "version_slug": os.environ.get('READTHEDOCS_VERSION') or '',
```

Comment:
Oooo... this looks promising. Won't ever work on local build but I don't think it matters.

Reply:
Exactly, it only matters to remove the line on latest and stable on RTD